etc/hexchat.profile

```diff
@@ -1,5 +1,6 @@
 # HexChat instant messaging profile
 noblacklist ${HOME}/.config/hexchat
+noblacklist /usr/lib/python2*
```

Comment:
I don't use hexchat, so I don't know what is really required.
I just thought other scripting languages would also be required to run plugins/addons.